prometheus-ksonnet/lib/prometheus-config.libsonnet

```diff
@@ -171,6 +171,17 @@
             replacement: '__param_$1',
           },
 
+          // Generic mapping of k8s label/annotation to prometheus labels.
```

Comment:
Perhaps mention the precise prefix requirement here?

E.g. "Map all K8s labels/annotations starting with 'prometheus.io/label-<name>' to Prometheus labels."